src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -522,7 +532,7 @@
     <channel-type id="meter_kwh">
         <item-type>Number</item-type>
         <label>Energy Consumption</label>
-        <description>Indicates the energy consumption (kWh)</description>
+        <description>Indicates the instantaneous energy consumption (kWh)</description>
```

Comment:
For me, this is wrong - you don't have instantaneous energy consumption measured in watt-hours. Energy consumption is measured in Wh -  but instantaneous power consumption is measures in watts.

I'm not sure how this links to the ZWave data definition and how that is applied to this channel, but fundamentally it's not correct to add instantaneous here and call it kWh.

Reply:
You are right. Togehter with energy consumption there is rarely used "instantaneous" but often "active" in opposite to the reactive and aparent energy consumption. So we can leave it as energy consumption or change to active energy consumption.